### DISCUSSION TITLE
Anyone using 22.11 yet in NixOS-WSL?

### DISCUSSION THREAD

**Discussion by webframp** [nix-community/NixOS-WSL#244] [General]
I know there's no official release yet for 22.11, but saw this change https://github.com/nix-community/NixOS-WSL/pull/180 and wondered if anyone is using 22.11 successfully yet on WSL?

I tried an upgrade (via channel add) and blew up my WSL setup, since it failed to start correctly and I have no idea how to debug that kind of thing with WSL. At least recovery is easy with nixos.

**Comment by SuperSandro2000** [comment]
People like me are using master on nixos-unstable on a daily basis but with flakes instead of channels.

**Reply by webframp** [reply]
Thanks for confirming, that sounds like the direction I'd like to go
